server/routes/v1/posts.js

```diff
@@ -0,0 +1,16 @@
+// modules
+const express = require('express');
+const router = express.Router();
+
+// controller
+const post = require('../../controllers/post');
+
+// middleware
+const token = require('../../middlewares/token');
+
+router.post('/post/create', token.validate, post.create)
+router.get('/post/:slug', post.getPostById)
```

Comment:
User should be logged in to view posts right? We need token validation here as well, no?

Reply:
> User should be logged in to view posts right?

All the posts are accessible to the member, regardless of the auth.

But they won't be able to comments on a post until they login.

Reply:
So, technically anybody on the internet can access this post as long as they have slug, correct?

Reply:
Right, after all it's a public site.

Reply:
That's actually good. It makes things a little transparent.